### ISSUE TITLE
v2.5.0 Error when retrieving streaming data

### ISSUE THREAD

**Issue by hydego17** [LuanRT/YouTube.js#248]
### Steps to reproduce

This happened inside Next.js API routes that I build, following the guide in the [Docs](https://github.com/LuanRT/YouTube.js#getstreamingdatavideo_id-options) to get video info and retrieve streaming data.

```js
const info = await yt.getBasicInfo('somevideoid');
const url = info.streaming_data?.formats[0].decipher(yt.session.player);
```

### Failure Logs
*When I tried to resolve it by upgrading `acorn` package, the error pointed out to `jintr` package instead.

```shell
SyntaxError: Unexpected token (1:52)
    at Parser.pp$4.raise (<project-path>/node_modules/acorn/dist/acorn.js:3459:15)
    at Parser.pp$9.unexpected (<project-path>/node_modules/acorn/dist/acorn.js:760:10)
    at Parser.pp$9.semicolon (<project-path>/node_modules/acorn/dist/acorn.js:737:68)
    at Parser.pp$8.parseVarStatement (<project-path>/node_modules/acorn/dist/acorn.js:1168:10)
    at Parser.pp$8.parseStatement (<project-path>/node_modules/acorn/dist/acorn.js:915:19)
    at Parser.pp$8.parseBlock (<project-path>/node_modules/acorn/dist/acorn.js:1236:23)
    at Parser.pp$5.parseFunctionBody (<project-path>/node_modules/acorn/dist/acorn.js:3285:24)
    at Parser.pp$8.parseFunction (<project-path>/node_modules/acorn/dist/acorn.js:1358:10)
    at Parser.pp$8.parseFunctionStatement (<project-path>/node_modules/acorn/dist/acorn.js:1058:17)
    at Parser.pp$8.parseStatement (<project-path>/node_modules/acorn/dist/acorn.js:903:19) {
  pos: 52,
  loc: Position { line: 1, column: 52 },
  raisedAt: 53
}
}
```

### Expected behavior

Retrieving stream data from video info should be working just fine.

### Current behavior

Its working fine when I tried to access info.with `getBasicInfo`, but when I tried to retrieve streaming data by invoking `info.streaming_data?.formats[0].decipher(yt.session.player)` the error happened.  

Maybe be related to decipher algorithm error?

### Version

Default

### Checklist

- [ ] I am running the latest version.
- [X] I checked the documentation and found no answer.
- [X] I have searched the existing issues and made sure this is not a duplicate.
- [X] I have provided sufficient information.

**Comment by LuanRT** [comment]
Yup, seems like signature decipher extraction is failing. For now, you can try changing the InnerTube client to `ANDROID`. 

Ex:
```ts
const info = await yt.getInfo('somevideoid', 'ANDROID');
console.log(info.streaming_data?.adaptive_formats?.[0].decipher(yt.session.player);
```

**Comment by hydego17** [comment]
@LuanRT  Well In production, I implement the request through proxy to create yt instance. Is there any downside if I switched the client to `ANDROID`?

**Comment by LuanRT** [comment]
Nope, should work just like the default `WEB` client. The only downside is that it doesn't have a few formats, but it's not a big deal.

**Comment by LuanRT** [comment]
Should be fixed now, see #249. You can use the default client again if you want/need.
